modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -90,7 +90,7 @@ controlPlane:
 
 ifeval::[{product-version} >= 4.4]
 a| [[provisioningNetworkInterface]]`provisioningNetworkInterface` |  | The name of the network interface on control plane nodes connected to the
-provisioning network.
+provisioning network. As of {product-title} 4.4 and later releases, this field is now optional due to the `bootMacAddress` field calculating the IP address for Ironic to bind to.
```

Comment:
```suggestion
provisioning network. In {product-title} 4.4 and later releases, this field is now optional due to the `bootMacAddress` field calculating the IP address for Ironic to bind to.
```

I would also specify the field you are talking about. It maybe helpful for translation purposes.

Reply:
Excellent point! I erroneously thought that it would be ok to skip the specific reference, as I defined the field in the prior sentence. I think you're absolutely right that it should be included, especially for translation purposes. Thank you so much for the review!